### PR TITLE
update Icon readme

### PR DESCRIPTION
**Pull request by chrismohr**

### CODE REVIEW

File: assets/icons/README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ These are the source SVG icons used to generate the corresponding React componen
 * Add prepared SVGs to this folder.
 * Run `npm run svg`.
 * Export the new icon from `src/components/Icon/icons/index.ts`.
+* Run `y run test:visual:component Icon` to generate a new reference image. ([Requires docker](https://github.com/microsoft/yamui#installation))
+* Run `npm run test:visual:approve` to approve the new reference image.
```
Comment (unindented) [inline assets/icons/README.md]: Do we need a similar thing for the `illustrations/` folder?

Comment (chrismohr) [reply assets/icons/README.md]: Yes, I think so.  I'll add one.

Comment (chrismohr) [reply assets/icons/README.md]: @unindented ok, I took a stab at it, feel free to nitpick/tune it, since I'm not too familiar with the process.

Comment (chrismohr) [reply assets/icons/README.md]: cc @Logrologist , since you're our new Illustration expert. :-)

**Pull request merged**